src/scrabble.js

```diff
@@ -1,5 +1,146 @@
-function scrabble() {
+const doubleArray = []
+const tripleArray = []
+const alphabet = [
+  'A',
+  'B',
+  'C',
+  'D',
+  'E',
+  'F',
+  'G',
+  'H',
+  'I',
+  'J',
+  'K',
+  'L',
+  'M',
+  'N',
+  'O',
+  'P',
+  'Q',
+  'R',
+  'S',
+  'T',
+  'U',
+  'V',
+  'W',
+  'X',
+  'Y',
+  'Z'
+]
 
+function scrabble(word) {
+  let score = 0
+  const check = wordCheck(word) // this function checks that the word only has acceptable characters
+  console.log(check) // this displays whether the word is valid
+  if (check === true) {
+    word = word.toUpperCase()
+    for (let i = 0; i < word.length; i++) {
+      const letter = word[i]
+      // console.log(letter) // this shows the individual letters been checked
+      score += scoring(letter)
+    }
+    if (doubleArray.length !== 0) {
+      console.log(doubleArray)
+      for (let i = 0; i < doubleArray.length; i++) {
+        const letter = doubleArray[i].toUpperCase()
+        score += scoring(letter) * 2
+      }
+    }
+    if (tripleArray.length !== 0) {
+      console.log(tripleArray)
+      for (let i = 0; i < tripleArray.length; i++) {
+        const letter = tripleArray[i].toUpperCase()
+        score += scoring(letter) * 3
+      }
+    }
+  }
+  return score
 }
 
+function scoring(letter) {
+  const l = letter
+  if (
+    l === 'A' ||
+    l === 'E' ||
+    l === 'I' ||
+    l === 'O' ||
+    l === 'U' ||
+    l === 'L' ||
+    l === 'N' ||
+    l === 'R' ||
+    l === 'S' ||
+    l === 'T'
+  ) {
+    return 1
+  } else if (l === 'D' || l === 'G') {
+    return 2
+  } else if (l === 'B' || l === 'C' || l === 'M' || l === 'P') {
+    return 3
+  } else if (l === 'F' || l === 'H' || l === 'V' || l === 'W' || l === 'Y') {
+    return 4
+  } else if (l === 'K') {
+    return 5
+  } else if (l === 'J' || l === 'X') {
+    return 8
+  } else if (l === 'Q' || l === 'Z') {
+    return 10
+  }
+}
```

Comment:
Nice use of else if statements. I used a switch statement for this portion, you should check them out and see if it's something you'd like to utilise in the future.